README.md

```diff
@@ -67,6 +67,29 @@ trkB.SetRotationFrequency(1 * time.Minute)
 trk, err := trkB.Build()
 defer trk.Close()
 ```
+Enabling Stats and Debugging Bucket Details
+To collect and inspect per-bucket statistics for debugging, set IncludeStats to true in your tracker config:
+
+```go
+cfg := &config.FairnessTrackerConfig{
```

Comment:
So users almost never directly construct the config object like this so this may give an impression that they have to hand-make one for including stats. Better show setting this to true on an existing config before creating the tracker.

Reply:
makes sense. updated. I see setIncludeStats can also be used after creating the tracker.